src/tests/youtube/Youtube_EME_001.test.js

```diff
@@ -6,6 +6,8 @@ import { AttachToLogs } from '../../commonMethods/remoteWebInspector'
 let url =
   'https://yt-dash-mse-test.commondatastorage.googleapis.com/unit-tests/2018.html?test_type=encryptedmedia-test&enablewebm=off&command=run'
```

Comment:
Should we keep this test in? I dont think that url works anymore right?

Reply:
Yes the URL doesn't work and this test is no more in master.